lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -559,131 +149,22 @@ module.exports = {
-            "no-regexp-named-capture-groups":
-                "RegExp named capture groups are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-regexp-s-flag":
-                "RegExp 's' flag is not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-regexp-unicode-property-escapes":
-                "RegExp Unicode property escapes are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-rest-spread-properties":
-                "Rest/spread properties are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-
-            //------------------------------------------------------------------
-            // ES2019
-            //------------------------------------------------------------------
-            "no-json-superset":
-                "'\\u{{code}}' in string literals is not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-optional-catch-binding":
-                "The omission of 'catch' binding is not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-
-            //------------------------------------------------------------------
-            // ES2020
-            //------------------------------------------------------------------
-            "no-bigint":
-                "Bigint literals are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-bigint-property-names":
-                "Bigint literal property names are not supported yet.",
-            "no-dynamic-import":
-                "'import()' expressions are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-optional-chaining":
-                "Optional chainings are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-nullish-coalescing-operators":
-                "Nullish coalescing operators are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-
-            //------------------------------------------------------------------
-            // ES2021
-            //------------------------------------------------------------------
-            "no-logical-assignment-operators":
-                "Logical assignment operators are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
-            "no-numeric-separators":
-                "Numeric separators are not supported until Node.js {{supported}}. The configured version range is '{{version}}'.",
+            "not-supported-till": [
+                "'{{ruleId}}' is not supported until Node.js {{supported}}.",
```

Comment:
nits: the `{{ruleId}}` is a bit confusing, maybe `{{feature}}` or something is better?

Reply:
Yeah, that would make sense!